internal/mirror/cmd/pull/validation.go

```diff
@@ -70,11 +73,12 @@ func validateImagesBundlePathArg(args []string) error {
 	if err != nil {
 		return fmt.Errorf("Read bundle directory: %w", err)
 	}
-	if len(dirEntries) > 0 {
-		return fmt.Errorf("%s is not empty, use --force to override", ImagesBundlePath)
+
+	if len(dirEntries) == 1 && dirEntries[0].Name() == ".tmp" {
```

Comment:
This would not work for empty directories
```suggestion
	if len(dirEntries) == 0 || (len(dirEntries) == 1 && dirEntries[0].Name() == ".tmp") {
```